blueocean-config/src/main/java/io/jenkins/blueocean/config/BlueOceanConfigStatePreloader.java

```diff
@@ -83,4 +88,22 @@ public String getStateJson() {
     private String getBlueOceanPluginVersion(){
         return Jenkins.getInstance().getPlugin("blueocean-web").getWrapper().getVersion();
     }
+
+    /**
+     * Returns true if the orgatizations are enabled on this instance
+     * 
+     * @return true if the organizations are enabled
+     */
+    public static boolean isOrganizationsEnabled() {
+        return organizationsEnabled;
+    }
+
+    /**
+     * Sets the organizations enabled/disabled flag
+     * 
+     * @param organizationsEnabled true to enable the organizations
+     */
+    public static void setOrganizationsEnabled(boolean organizationsEnabled) {
```

Comment:
Yes, I just thought there was going to be a rewrite of that part (per the comment) that's why I went for a simpler solution. Will do the extension.